packages/client/hmi-client/src/components/operator/tera-operator-menu.vue

```diff
@@ -1,25 +1,27 @@
 <template>
-	<Button
-		v-bind="$attrs"
-		type="button"
-		size="small"
-		@click="showMenu"
-		icon="pi pi-plus"
-		aria-haspopup="true"
-		aria-controls="overlay_menu"
-		severity="secondary"
-	/>
-	<Menu
-		ref="menu"
-		id="overlay_menu"
-		:model="menuItems"
-		:popup="true"
-		@focus="emit('menu-focus')"
-		@blur="emit('menu-blur')"
-	/>
+	<aside>
```

Comment:
just surrounded with a parent `aside` so that v-show would work

Reply:
and `<aside>` is a great choice. A perfect one would have been `<nav>`, but I'm splitting hairs here. Do not change anything! It's beautiful and precious as it is.